integration-test/pom.xml

```diff
@@ -80,346 +87,190 @@
+                        <configuration>
+                            <basedir>${project.build.directory}/template-node</basedir>
+                            <regex>true</regex>
+                            <commentsEnabled>false</commentsEnabled>
+                            <includes>
+                                <include>datanode/conf/iotdb-env.sh</include>
+                                <include>confignode/conf/confignode-env.sh</include>
+                                <include>datanode/conf/iotdb-env.bat</include>
+                                <include>confignode/conf/confignode-env.bat</include>
+                            </includes>
+                            <replacements>
+                                <replacement>
+                                    <token>#HEAP_NEWSIZE=.*</token>
+                                    <value>HEAP_NEWSIZE="${integrationTest.nodeNewHeapSize}"</value>
+                                </replacement>
+                                <replacement>
+                                    <token>#MAX_HEAP_SIZE=.*</token>
+                                    <value>MAX_HEAP_SIZE="${integrationTest.nodeMaxHeapSize}"</value>
+                                </replacement>
+                                <replacement>
+                                    <token>@REM set HEAP_NEWSIZE.*</token>
+                                    <value>set HEAP_NEWSIZE="${integrationTest.nodeNewHeapSize}"</value>
+                                </replacement>
+                                <replacement>
+                                    <token>@REM set MAX_HEAP_SIZE=.*</token>
+                                    <value>set MAX_HEAP_SIZE="${integrationTest.nodeMaxHeapSize}"</value>
+                                </replacement>
+                            </replacements>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- Run integration tests -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>integration-test</id>
+                        <goals>
+                            <goal>integration-test</goal>
+                        </goals>
+                        <configuration>
+                            <skipTests>false</skipTests>
+                            <groups>${integrationTest.includedGroups}</groups>
+                            <excludedGroups>${integrationTest.excludedGroups}</excludedGroups>
+                            <useSystemClassLoader>false</useSystemClassLoader>
+                            <parallel>${integrationTest.parallelMode}</parallel>
+                            <useUnlimitedThreads>true</useUnlimitedThreads>
+                            <threadCount>1</threadCount>
```

Comment:
This `threadCount` line can be removed.  Setting `useUnlimitedThreads` to `true`  will disable this setting.

Reply:
Fixed.